src/expressions.rst

```diff
@@ -659,31 +659,56 @@ Block Expressions
 A :t:`block expression` is an :t:`expression` that sequences :t:`[expression]s`
 and :t:`[statement]s`.
 
+:dp:`fls_gOxD5xdYLaQG`
+A :t:`named block expression` is a :t:`block expression` with a :t:`label`.
+
+:dp:`fls_ZEX9YIxHlnVr`
+A :t:`named block expression` shall appear within a :t:`block expression`.
```

Comment:
Why is this?

Reply:
After experimenting, this doesn't make sense indeed.